lib/rails_response_dumper/runner.rb

```diff
@@ -112,6 +112,12 @@ def run_dumps
               }
             }
 
+            # request.headers includes nonstandard internal data, some of which also lacks default deterministic
+            # serialization. Here we only want CGI standard and HTTP variables.
+            dump[:request][:env].delete_if do |key, _|
```

Comment:
The 2nd arg goes unused.

```suggestion
            dump[:request][:env].delete_if do |key|
```